starter/source/output/message/starter_message_description.txt

```diff
@@ -1423,6 +1423,17 @@
        
 
 
+/MESSAGE/132/TITLE
+** ERROR IN DENSITY FUNCTION DEFINITION
+/MESSAGE/132/DESCRIPTION
+   -- MATERIAL ID: %d
+   -- MATERIAL TITLE: %s
+   -- FUNCTION ID: %d
+   -- NEGATIVE OR NULL DENSITY VALUE %f POINT NUMBER %d
+   %i1="/FUNCT"  
```

Comment:
@Schwoertzig 

I think %i1 should be "/MAT" , & %i2 "/FUNCT"

Reply:
@sebastienVilleneuve ,
correct ! It is now updated. thanks.